lib/pbxProject.js

```diff
@@ -1593,6 +1615,9 @@ function pbxShellScriptBuildPhaseObj(obj, options, phaseName) {
     obj.outputPaths = options.outputPaths || [];
     obj.shellPath = options.shellPath;
     obj.shellScript = '"' + options.shellScript.replace(/"/g, '\\"') + '"';
+    if(options.showEnvVarsInLog != null && !options.showEnvVarsInLog) {
```

Comment:
I personally prefer strict equal checks. I know that != covers both `null` or `undefined` but I personally avoid it because it is not that obvious and forces contributors to know all the rules

```suggestion
    if(options.showEnvVarsInLog !== null && options.showEnvVarsInLog !== undefined  && !options.showEnvVarsInLog) {
```